### ISSUE TITLE
Testing gtp-U connection with free5gc upf 

### ISSUE THREAD

**Issue by mrramachari** [wmnsk/go-gtp#272]
Hi,I'm new to golang and building a core network and i want to use your library to establish user plane connection with free5gc UPF. 
I'm finding it hard to follow the Readme instructions given, It will be helpful if you could give me some examples to follow.   

I Have established session on the upf side and I would like to test it with your gtpv1 package.

I have tested u-conn_test.go and im unable to handle the response received from the upf gtp modules.

Thanks in advance.

**Comment by wmnsk** [comment]
In general, use something instead of `_test.go` which are basically just unit tests. For more help I need the details on what you did with your code, but please remember that this is the library for Go developers in the first place.

**Comment by mrramachari** [comment]
Thank you for your response,I will go through all the examples and try to understand it as you suggested and will then proceed with my code.
